sdk/python/feast/feature_store.py

```diff
@@ -549,13 +581,15 @@ def _plan(
             desired_repo_contents.feature_views,
             desired_repo_contents.on_demand_feature_views,
             desired_repo_contents.request_feature_views,
+            desired_repo_contents.stream_feature_views,
```

Comment:
given that you're changing the `feast apply` flow I think we should also add a test that SFVs can be applied properly - maybe something similar to `test_feature_service_apply.py`?

Reply:
Added test.